src/routes/api.js

```diff
@@ -1,5 +1,6 @@
 import express from 'express';
 const router = express.Router();
+import { nanoid } from 'nanoid';
 
 router.get('/', (_, res) => {
```

Comment:
maybe the home route is in a separate file like `home` or something, since its not an API route

Reply:
Good point. I separated the home route out from the api routes (commit is below 👇 )